extensions/ql-vscode/src/remote-queries/view/RemoteQueries.tsx

```diff
@@ -92,6 +93,39 @@ const QueryInfo = (queryResult: RemoteQueryResult) => (
   </>
 );
 
+const Failures = (queryResult: RemoteQueryResult) => {
+  if (queryResult.analysisFailures.length === 0) {
+    return <></>;
+  }
+  return (
+    <>
+      <VerticalSpace size={3} />
+      <Flash
+        variant="danger"
+        // Currently only works for dark mode (potential bug with Flash component)
+        // https://primer.style/react/Flash
+        sx={{
```

Comment:
Does this not work in light mode? I do think it is important to work in light mode.

Do you have a link to the bug report? This way we can know if this gets fixed in the future.

Reply:
Fixed in https://github.com/github/vscode-codeql/pull/1159/commits/9bf84bdd2cd7e9081201c7d9e0600d76d5cd73c5